packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -11,6 +11,11 @@ import type {
 	CloudflareEnvironmentOptions,
 	CloudflareDevEnvironment,
 } from './cloudflare-environment';
+import {
+	getModuleFallbackCallback,
+	type ResolveIdFunction,
```

Comment:
Is it OK if we keep type imports separate and below the other imports? I just find it easier to scan.

Reply:
Sure, I don't have a particular preference

Sorry for not picking up the consistency you're introduced of using the top-level version 🙇 

PS: as a followup I can add this eslint rule: https://typescript-eslint.io/rules/consistent-type-imports/#fixstyle to make sure we keep all imports consistent in this regard

Reply:
https://github.com/cloudflare/workers-sdk/blob/main/.prettierrc

Reply:
I've updated the code here 🙂 

I'll resolve this conversation if that's ok we can then follow up separately on formatting/linting 🙂